axon-server-connector/src/main/java/org/axonframework/axonserver/connector/util/FlowControllingStreamObserver.java

```diff
@@ -56,24 +57,46 @@
     public FlowControllingStreamObserver(StreamObserver<T> wrappedStreamObserver, AxonServerConfiguration configuration,
```

Comment:
Nit: Would you mind having this constructor reference the new one you've created through `this(...)`? This would minimize potential discrepancies between both constructors.